SetGame/SetGameTests/SetGameTests.swift

```diff
@@ -0,0 +1,144 @@
+//
+//  SetGameTests.swift
+//  SetGameTests
+//
+//  Created by Nogah Melamed Cohen on 15/10/2019.
+//  Copyright © 2019 Nogah Melamed Cohen. All rights reserved.
+//
+
+import XCTest
+@testable import SetGame
+
+class SetGameTests: XCTestCase {
```

Comment:
The tests are excellent, I really enjoyed going through this test suite. Also I run the tests and was happy to see that it's all green. It makes me happy everytime I do it :)